storybook/stories/SideMenu.stories.js

```diff
@@ -23,21 +23,32 @@ storiesOf('SideMenu', module)
 }))
 .add('custom', () => ({
   components: { Sider },
-  template: `<Sider :items="items">
-      <template v-slot:general="{ item }">
-        <a :href="item.href" class="link-secondary">        
-          <i class="chevron right"></i>
-          <span>[Custom]-general</span>
-        </a>  
-      </template>
+  template: `
+    <div style="min-height:1000px;background:#eee">
```

Comment:
could you just put a style like this
```
    min-height: 5000px;
    background: linear-gradient(0deg, rgba(0,0,0, .5), transparent);
```
Like that we see the impact of the scroll and we can remove all the text that you put

Reply:
Done